src/eth/utils.ts

```diff
@@ -0,0 +1,11 @@
+import type { InternalAccountType } from '../internal';
+import { EthAccountType } from './types';
+
+/**
+ * Checks if the given internal account is of the EVM account type.
```

Comment:
Maybe we should adapt the doc here, since we are now using the account type exclusively?